.env.example

```
@@ -58,10 +54,10 @@ TF_SHELL_PORT=3103
 TF_DESKTOP_PORT=3104
 TF_STATIC_PORT=8080
 
-# ===== API URLs =====
-VITE_API_URL=http://localhost:${TF_API_PORT:-5000}/api
-REACT_APP_API_GATEWAY=http://localhost:${TF_API_PORT:-5000}
-ASPNETCORE_URLS=http://localhost:${TF_API_PORT:-5000}
+# ===== API URLs (consume from ports above) =====
+VITE_API_URL=http://localhost:${TF_API_PORT:-5046}/api
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The VITE_API_URL key is not assigned properly

(SubstitutionKey)